docs/doc/10-deploy/09-upgrade/90-upgrade-0.8-to-0.9.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Upgrade Databend-query from 0.8 to 0.9
+sidebar_label: Upgrade Databend-query from 0.8 to 0.9 
+description:
+    Meta data migration when upgrading databend-query from 0.8 to 0.9
+---
+
+This topic explains how to migrate metadata when upgrading databend-query from 0.8 to 0.9
+
+Databend-query-0.9 introduces incompatible changes in metadata, these metadata has to be manually migrated.
+Databend provides a program for this job: `databend-meta-upgrade-09`, which you can find in a [release package](https://github.com/datafuselabs/databend/releases) or can be built from source.
+
+`databend-meta-upgrade-09` upgrades metadata written by databend-query before 0.9 to a databend-query-0.9 compatible format.
+It loads metadata from a `raft-dir` and upgrade `TableMeta` record to 0.9 compatible version and write them back.
+Both raft-log data and state-machine data will be converted.
+
+# Usage
+
+- Shut down all databend-meta processes.
+
+- Before proceeding, backup your data. See: [Backup and resotre metadata](../06-metasrv/30-metasrv-backup-restore.md).
+
+- Download `databend-meta-upgrade-09` from [Databend release](https://github.com/datafuselabs/databend/releases)
```

Comment:
If we introduce the new version in the future, will the binary number of upgrade be increased?

Reply:
Yes and I hope so.

It would be convenient if there is only one upgrade program that always upgrades metadata to the latest version.